client/my-sites/importer/newsletter/importer.tsx

```diff
@@ -68,6 +80,12 @@ export default function NewsletterImporter( { siteSlug, engine, step }: Newslett
 		from: fromSite,
 	} );
 	const Step = steps[ stepIndex ] || steps[ 0 ];
+
+	let title = 'Import your newsletter';
```

Comment:
I know that these strings were already untranslated before you made any changes (!), but is it intentional to keep it this way?

Reply:
Yes this code is under a feature flag and the strings might change before it goes live. We are planning on translating them once all the strings are finalized.